packages/engine/src/worker/runner/javascript/mod.rs

```diff
@@ -433,133 +618,233 @@ fn array_to_user_warnings(array: mv8::Value<'_>) -> Vec<UserWarning> {
     }]
 }
 
-fn get_js_error(_mv8: &MiniV8, return_val: &mv8::Object<'_>) -> Option<Error> {
-    if let Ok(errors) = return_val.get("user_errors") {
-        if !matches!(errors, mv8::Value::Undefined) && !matches!(errors, mv8::Value::Null) {
-            let errors = array_to_user_errors(errors);
+fn get_js_error<'s>(
+    scope: &mut v8::HandleScope<'s>,
+    return_val: v8::Local<'s, v8::Object>,
+) -> Option<Error> {
+    tracing::debug!("Collecting JS errors");
```

Comment:
These don't seem like useful debug lines, perhaps better as trace. But they will spam the logs otherwise.

Reply:
Addressed in https://github.com/hashintel/hash/pull/414/commits/ae2501dcb17238e72eefc7eae179b72f0c899b82